sqllogictest-engines/src/mysql.rs

```diff
@@ -0,0 +1,102 @@
+use std::process::Command;
+use std::time::Duration;
+
+use async_trait::async_trait;
+use sqllogictest::{DBOutput, DefaultColumnType};
+use sqlx::{mysql::MySqlPoolOptions, Column, Row, TypeInfo};
+
+type Result<T> = std::result::Result<T, sqlx::Error>;
+
+/// Connection configuration. This is a re-export of [`sqlx::mysql::MySqlConnectOptions`].
+pub type MySqlConfig = sqlx::mysql::MySqlConnectOptions;
+
+/// MySQL engine based on the client from [`sqlx::mysql`].
+pub struct MySql {
+    pool: sqlx::Pool<sqlx::MySql>,
```

Comment:
I'm just kind of concerned whether it's good to introduce the dependency on `sqlx` solely for MySQL. It's quite a heavy dependency, whilst we directly depend on `tokio-postgres` for the Postgres engine.

If we find `sqlx` can simplify and unify the implementation for multiple engines, then it's good and we'd better completely switch to it. Otherwise, `mysql-async` seems to be a more lightweight choice.

Reply:
Thanks for your suggestion. It seems that switching to sqlx also cannot avoid the different handling required for various storage engines. I will try a lighter implementation later.